### ISSUE TITLE
Support InputLayer for sequential models

### ISSUE THREAD

**Issue by dguest** [lwtnn/lwtnn#109]
Related to #108: it seems that some newer sequential keras models have an `InputLayer` that I had to remove to get a sensible conversion. I can probably just ignore this.

**Comment by QuantumDancer** [comment]
Hi,
I've noticed the same problem when upgrading from keras 2.3 to 2.4.

My solution was to add the entry `'inputlayer'` to the `skip_layers` list in `converters/keras_layer_converters_common.py`, so that I don't have to modify the input to the converter script by hand.

**Comment by dguest** [comment]
Hi @QuantumDancer, good to know it was so easy. Sorry I dropped this report for a while. Feel free to make a MR if you think everything is working.

**Comment by QuantumDancer** [comment]
Hi @dguest, sorry I kind of forgot to follow up on this. I have this fix in use for quite some time and the output of lwtnn seems reasonable. I've created a MR with the fix described above.

**Comment by tvoigtlaender** [comment]
Hi,
I also encountered the same problem and have been using the fix described by @QuantumDancer.

**Comment by dguest** [comment]
Yeah, sorry I haven't merged #117, it's breaking the CI tests and I haven't had time to look into why. That will need to be fixed before we merge it in.

**Comment by QuantumDancer** [comment]
I just had a look at the CI. It looks like adding this to the common list of layers to ignore messes with the conversion of functional networks. As I am only using sequential ones I didn't run into this issue.

I've changed the MR so that layers of type `inputlayer` are only skipped for sequential networks. The CI checks pass now.

Right now I just put the definition of the layers which should be additionally skipped for sequential networks directly into the function where it used. Please let me know if I should put this somewhere else. I didn't want to put this into the `keras_layer_converters_common.py` file, as this is not common to both sequential and functional networks.

**Comment by dguest** [comment]
Thanks. I guess this should be fixed now, correct?

**Comment by QuantumDancer** [comment]
Yes!

**Comment by tvoigtlaender** [comment]
Thank you for the quick response.
